### DISCUSSION TITLE
Figure 1: is it too early / needed in main text

### DISCUSSION THREAD

**Discussion by hyanwong** [tskit-dev/what-is-an-arg-paper#194] [General]
I wonder whether figure 1 meets our requirements of making the paper (almost) immediately understandable from the figures. I find it quite an involved picture, which takes a while to get your head around. For this reason, I didn't use it in the ProbGen talk I gave. In addition, I feel as if going into careful detail of the Hudson algorithm in (basically) the first section rather defeats points 1 and 2 of https://github.com/tskit-dev/what-is-an-arg-paper/discussions/20. I discussed this with @jeromekelleher but we didn't come to any obvious conclusions.

Starting out with something like figure 2 gives a much easier visual path into the paper, and makes it a tighter argument. But placing the figure / discussion entirely into supplementary seems rather drastic. I wonder if there are other options which would work well?

**Comment by jeromekelleher** [comment]
I tend to agree with this. We'll need to make a pass through the text and rejig, now that most of the graphical content is in. I'll hopefully get back to writing later in the week (still digging out of postponed admin pile)

**Comment by JereKoskela** [comment]
I think we do need something like the exposition with the notation for lineages, children & parents, the various rates, etc., even if we're targeting people who in principle know that stuff. We need to introduce the our notation, and I think we'll lose (even experienced) people right at the start if we don't give a brief but precise account of exactly what we're talking about.  Perhaps it could be trimmed down though.

On the other hand, it stands out to me that we don't refer to Figure 1 anywhere in the text at the moment. I have no objection for it to be shifted to a supplement as something a reader can look up if they want a concrete illustration of the notation and terminology.
